source/_integrations/fireservicerota.markdown

```diff
@@ -83,7 +83,7 @@ This entity reflects the duty you have scheduled, the value can be `on` = on dut
 ### Incident response switch
 
 With this switch you can respond to an incident, either by manually controlling the switch via the GUI, or by using an automation action.
-It gets reset to `unknown` value with every incident received. Switching it to `on` means you send a response acknowledgement, switching it back `off` sends a response rejected.
+It gets reset to `unknown` value with every incident received. Switching it to `on` means you send a response acknowledgment, switching it back `off` sends a response rejected.
```

Comment:
Both are valid but the proposal seems to be the preferred spelling for en-us. So ok.

Reply:
Oh, wild. I could have sworn that it was the same for en-us and en-uk. I've been trying to account for both when it comes to integrations specifically.

Since this integration in particular is for a UK-based service (or Netherlands) it would seem more appropriate to not change this spelling since that is the target audience.

Reply:
I am a UK person so often these differences jump out, but the [official language for the documentation is American English](https://developers.home-assistant.io/docs/documenting/standards/#general-documentation).

So what you put in the PR is more correct.